lib/configs/routes.dart

```diff
@@ -1,6 +1,5 @@
-import 'package:flutter/widgets.dart';
-import 'package:survey/screens/landing/landing_screen.dart';
+part of 'package:survey/gen/configs.gen.dart';
 
-final Map<String, WidgetBuilder> routes = {
+final Map<String, WidgetBuilder> _routes = {
   "/": (_) => LandingScreen(),
 };
```

Comment:
for the future usage (navigate by `pushNamed`), I would recommend creating const variable for route name instead of typing name manually 🙏

Reply:
Agree!! Let's me update it 👍